src/03-deals/deals.apib

```diff
@@ -219,6 +222,7 @@ Update a deal.
                 + id: `09e5d75f-f817-4872-9723-57fbb8ff710d` (string, required)
             + contact_person_id: `b38ebb9b-6e46-4bf4-a1e2-af747d6b64ae` (string, optional)
         + title: `Interesting business deal` (string, optional)
+        + summary: `Additional information` (string, optional)
```

Comment:
Q: how can an integrator clear the summary for an existing deal?

Reply:
Good point so I guess that I need to allow empty strings after all.